profiles/seccomp/default.json

```diff
@@ -474,7 +474,8 @@
 		},
 		{
 			"names": [
-				"sync_file_range2"
+				"sync_file_range2",
+				"swapcontext"
```

Comment:
Just an observation (and for other reviewers); this adds the syscall conditionally for `ppc64le` only; the default profile currently doesn't have `ppc` (32-bit ppc), so it won't be available on those architectures.

I don't think we need to take 32-bit ppc into account, as we don't test/support that architecture, and it's a pretty niche use (I think there's some people doing their own custom builds of dockerd for 32-bit ppc, but not sure)